lib/core/src/lib/breadcrumb/components/breadcrumb/breadcrumb.component.scss

```diff
@@ -0,0 +1,50 @@
+ol {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    flex-wrap: wrap;
+    min-height: 40px;
+}
+
+.adf-breadcrumb__item-wrapper {
+    display: flex;
+    align-items: center;
+}
+
+.adf-breadcrumb__chevron {
+    background-image: url("breadcrumb-chevron.svg");
```

Comment:
you probably don't need that if using material icon

Reply:
This one is a custom one, that is why we using this one.